generator/internal/scaffold_generator.cc

```diff
@@ -149,11 +214,77 @@ void MakeDirectory(std::string const& path) {
 #endif  // _WIN32
 }
 
+void GenerateMetadata(
+    std::map<std::string, std::string> const& vars,
+    std::string const& output_path,
+    google::cloud::cpp::generator::ServiceConfiguration const& service,
+    bool allow_placeholders) {
+  auto const destination = [&]() {
+    MakeDirectory(output_path);
+    auto d = output_path + "/" + LibraryPath(service.product_path());
+    MakeDirectory(d);
+    auto l = vars.find("service_subdirectory");
+    if (l == vars.end()) return d;
+    if (l->second.empty()) return d;
+    d += l->second;
+    d += "/";
```

Comment:
I don't think this is necessary. `service_subdirectory` should always end in a slash.

https://github.com/googleapis/google-cloud-cpp/blob/c435bc9ac0cc6dc26833164a3f28e82c111eced4/generator/internal/scaffold_generator.cc#L82

Reply:
Removed.